### DISCUSSION TITLE
Can I change directional light and make shadow on other model

### DISCUSSION THREAD

**Discussion by man44587** [google/model-viewer#4568] [Q&A]
I want to set directional light to in front of camera not above of medel. Can I do it ?

**Comment by elalish** [comment]
Yes, you'll need to paint an HDR environment image. A single bright pixel is equivalent to a directional light. However, our camera moves relative to the enivronment by default, do if you need it to be fixed to the camera always, that probably won't work (you could manage in yaw, but not pitch).

**Reply by man44587** [reply]
Can I change direction of shadow?
![image](https://github.com/google/model-viewer/assets/44811963/0aefab67-0fe3-43e1-a482-a86dc26b1af5)

**Reply by elalish** [reply]
No, we don't support cast shadows. For that you'll need to fork us and add what you need to our internal Three.js scene.

**Reply by henrik-dons** [reply]
Hello, are there plans to include cast shadows in the future?
